.travis.yml

```diff
@@ -26,30 +26,34 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=3.4
+  matrix: SYMFONY_VERSION=^4@dev
```

Comment:
does that mean the default symfony version we test with is version 4? i guess thats fine once symfony 4 is released - which will hopefully be soon.

Reply:
not really, is just one entry in the matrix, one that could fail, cause failing vars should occur in the matrix once (that is how i learned it from travis docu)